test/helpers/content_helper_test.rb

```diff
@@ -461,7 +461,18 @@ class ContentHelperTest < ActionView::TestCase
     assert_equal excepted_response, response
   end
 
+  test 'render_api_namespace_resource_index - properly render using time_ago_in_words helper' do
```

Comment:
this test isn't sound, you shouldn't be hard coding `@api_resource_2.created_at` directly. You should derive the resources from the parent namespace. Eg: 

`@api_namespace.api_resources.first`

Reply:
Will change this. Thank you for reminding.